app/index.html

```diff
@@ -247,8 +247,10 @@ <h1>About Myself</h1>
       </div>
       <div class="card-description about-body">
         <p>
-          Hello I'm a software developer! I can help you build a product,
-          feature or website. Look through some of my work and experience! If
```

Comment:
- [ ] Please contact me section should have a background that matches the design the button should be centered in the screen same as the forms and the text please make sure that you have the margins and padding defined in the design 
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/9d6a0b1c-38fc-4b30-9dd1-383ff807a4de)
![image](https://github.com/Amaramhdi1112/portfolio-amar-muha/assets/74744787/97debda2-15ff-436e-a75b-4dbf89b9a45b)